modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java

```diff
@@ -51,8 +51,12 @@ public static List<Map<String, Object>> getCodeEditorElements(
 					ListUtil.filter(
 						_objectFieldLocalService.getObjectFields(
 							objectDefinitionId),
-						objectField -> !objectField.compareBusinessType(
-							ObjectFieldConstants.BUSINESS_TYPE_AGGREGATION)),
+						objectField ->
```

Comment:
LPS-165565 Add condition to send formula fields to validation and actions -> Could you please rewrite this commit message because this code is doing the opposite of the commit message?

Reply:
makes sense. thx

Reply:
Hey, @marcelabc. I will remove this commit because I'm updating this logic in another pull request and the commit message is right there.